package.json

```diff
@@ -9,8 +9,8 @@
     "preversion": "npm run lint",
     "build": "tsc",
     "clean": "rimraf lib",
-    "format": "eslint --fix '{src,example}/**/*.{ts,tsx}'",
-    "lint": "eslint '{src,example}/**/*.{ts,tsx}'",
+    "format": "eslint --fix {src,example}/**/*.{ts,tsx}",
```

Comment:
I think we should let eslint to the globbing, no?

Reply:
reverted